### PR TITLE
Feat/815 auction conditions

### LINKED ISSUE #815: Show auction entry and exit conditions
Somewhere obvious in Console 2 (maybe header), Show:
- Why is this market in auction
- Why/When did it go into auction
- What are the conditions for it exiting
- explanation of auction on vega (link to docs) 

note: this was created after https://github.com/vegaprotocol/frontend-monorepo/issues/656#issuecomment-1172211912

### PR DESCRIPTION
**Pull request by notbot00**
# Related issues 🔗

Closes #815 

# Description ℹ️

Add info in a tooltip for the market trading mode display.

# Demo 📺

Screenshot 2022-08-05 at 13 00 05
Screenshot 2022-08-05 at 13 00 19
Screenshot 2022-08-05 at 13 00 27
Screenshot 2022-08-05 at 13 00 42

### CODE REVIEW

File: apps/trading/components/trading-mode-tooltip/trading-mode-tooltip.tsx [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+
+  return grid;
+};
+
+type TradingModeTooltipProps = {
+  market: Market_market;
+};
+
+export const TradingModeTooltip = ({ market }: TradingModeTooltipProps) => {
+  console.log(market);
```
Comment (notbot00) [inline apps/trading/components/trading-mode-tooltip/trading-mode-tooltip.tsx:118]:
```suggestion

```

File: apps/trading/components/trading-mode-tooltip/trading-mode-tooltip.tsx [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+  market: Market_market;
+};
+
+export const TradingModeTooltip = ({ market }: TradingModeTooltipProps) => {
+  switch (market.tradingMode) {
+    case MarketTradingMode.Continuous: {
+      return (
+        <>
+          {t(
+            'This is the standard trading mode where trades are executed never orders are received.'
```
Comment (mattrussell36) [inline apps/trading/components/trading-mode-tooltip/trading-mode-tooltip.tsx:123]: chore (blocking): This reads odd, typo?

**Review by mattrussell36 — CHANGES_REQUESTED** [review]

File: apps/trading/components/trading-mode-tooltip/trading-mode-tooltip.tsx (hunk 2 above)
Comment (notbot00) [inline apps/trading/components/trading-mode-tooltip/trading-mode-tooltip.tsx:123]:
```suggestion
            'This is the standard trading mode where trades are executed whenever orders are received.'
```

Comment (notbot00) [reply apps/trading/components/trading-mode-tooltip/trading-mode-tooltip.tsx]: thanks for the catch!

**Review by mattrussell36 — APPROVED** [review]

**Review by MadalinaRaicu — APPROVED** [review]

**Review by sam-keen — APPROVED** [review]

**Pull request merged**